os/fs/smartfs/smartfs.h

```diff
@@ -347,6 +347,9 @@ struct smartfs_mountpt_s {
 	uint8_t *fs_chunk_buffer;
 #endif
 	uint8_t fs_rootsector;		/* Root directory sector num */
+#ifndef CONFIG_UTC
+	uint32_t entry_seq;
```

Comment:
I though you had suggested that smartfs_entry_s structure will hold sequence number.
The sequence number for that particular entry.